content/en/agent/faq/certificate_verify_failed-error.md

```diff
@@ -19,32 +19,34 @@ Datadog has published a [Python script][1] that queries your Datadog account for
 
 1. Download the [Python script][1].
 2. Run it in your local terminal or shell. The script works with Python 2 and 3.
-Datadog US: `python find_agents_with_connectivity_problems.py --api-key API_KEY --application-key APPLICATION_KEY --site us`  
-Datadog EU: `python find_agents_with_connectivity_problems.py --api-key API_KEY --application-key APPLICATION_KEY --site eu`  
+Datadog US: `python find_agents_with_connectivity_problems.py --api-key API_KEY --application-key APPLICATION_KEY --site us`
+Datadog EU: `python find_agents_with_connectivity_problems.py --api-key API_KEY --application-key APPLICATION_KEY --site eu`
 3. Find the CSV output in `hosts_agents.csv`.
 
-Get the API and application key [from your account settings page][4]. If you are using the EU site, see your [EU account settings page][5].
+Get the API and application key [from your account settings page][2]. If you are using the EU site, see your [EU account settings page][3].
```

Comment:
This looks correct to me here, but the rendered page has the US url on the `EU account settings page` link

Reply:
The region selector in the upper right controls US/EU site links.

Reply:
ok, I'll simply remove the mention of the EU site and link

Reply:
@jeremy-lq in that case, we should remove `If you are using the EU site, see your [EU account settings page][3].`?